modules/tide_site/js/quick_exit.js

```diff
@@ -0,0 +1,31 @@
+(function ($, Drupal) {
+
+    'use strict';
+  
+    Drupal.behaviors.siteQuickExitFields = {
+      attach: function (context, settings) {
+        var parent = ($('.taxonomy-term-sites-form  #edit-parent'));
```

Comment:
Instead of using parent, which is a common jQuery method, use a more descriptive variable name like parentField to avoid confusion.